fluent.bundle/fluent/bundle/types.py

```diff
@@ -0,0 +1,352 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import warnings
+from datetime import date, datetime
+from decimal import Decimal
+
+import attr
+import pytz
+from babel.dates import format_date, format_time, get_datetime_format, get_timezone
+from babel.numbers import NumberPattern, parse_pattern
+
+FORMAT_STYLE_DECIMAL = "decimal"
+FORMAT_STYLE_CURRENCY = "currency"
+FORMAT_STYLE_PERCENT = "percent"
+FORMAT_STYLES = set([FORMAT_STYLE_DECIMAL,
```

Comment:
I'll go with `_OPTIONS` because I also had `DATE_STYLE_OPTIONS` and `TIME_STYLE_OPTIONS`, and `_SET` ties you to a `set` when it's possible another container object might be more appropriate.